bindinfo/handle.go

```diff
@@ -150,6 +157,21 @@ func (h *BindHandle) AddBindRecord(sctx sessionctx.Context, is infoschema.InfoSc
 	if err != nil {
 		return err
 	}
+
+	br := h.GetBindRecord(parser.DigestHash(record.OriginalSQL), record.OriginalSQL, record.Db)
+	var duplicateBinding string
+	if br != nil {
+		binding := br.FindBinding(record.Bindings[0].id)
+		if binding != nil {
+			// There is already a binding with status `Using` or `PendingVerify`, we could directly cancel the job.
+			if record.Bindings[0].Status == PendingVerify {
+				return nil
+			}
+			// Otherwise, we need to remove it before insert.
+			duplicateBinding = binding.BindSQL
```

Comment:
If the existing binding is in `Using` status, we would remove it and re-insert? why?

Reply:
It is only for update the `update-time` field. Although we could also just ignore it here.